rosdep/python.yaml

```diff
@@ -186,6 +186,16 @@ exhale-pip:
   ubuntu:
     pip:
       packages: [exhale]
+fastapi-pip:
```

Comment:
Pip-only keys should be of the form:

`python3-fastapi-pip`.  The same goes for the key below.  Also be sure to put them in alphabetical order once you've changed it.